twake/frontend/src/app/scenes/Apps/Messages/Input/Parts/AttachedFiles.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+import MessageEditorsManager from 'app/services/Apps/Messages/MessageEditors';
+import File from 'components/Drive/File.js';
+import '../Input.scss';
+
+export default (props: { channelId: string; threadId: string }) => {
+  const editorManager = MessageEditorsManager.get(props.channelId);
+  editorManager.useListener(useState);
+
+  return (
+    (editorManager.filesAttachements[props.threadId || 'main'] !== undefined && (
+      <div className="attached-files-container small-y-margin">
+        {editorManager.filesAttachements[props.threadId || 'main'].map((id: string) => {
+          return (
+            <File
+              mini
+              key={id}
+              data={{ id: id }}
+              notInDrive={true}
+              removeIcon
+              removeOnClick={(e: any) => {
+                e.stopPropagation();
```

Comment:
Did you check that the preview was disabled when clicking outside the X button in this specific case ?
When the props "removeIcon" is true, I want the preview to be disabled.

Reply:
Done :+1: